update.sh

```diff
@@ -82,23 +75,26 @@ for dolibarrVersion in ${DOLIBARR_VERSIONS[@]}; do
   if [ ${DOCKER_BUILD} -eq 1 ]; then
     docker tag tuxgasy/dolibarr:${currentTag} tuxgasy/dolibarr:${dolibarrVersion}
     docker tag tuxgasy/dolibarr:${currentTag} tuxgasy/dolibarr:${dolibarrMajor}
+    if [ "${dolibarrVersion}" = ${DOLIBARR_LATEST_TAG} ]; then
+      docker tag tuxgasy/dolibarr:${currentTag} tuxgasy/dolibarr:latest
+    fi
   fi
   if [ ${DOCKER_PUSH} -eq 1 ]; then
     docker push tuxgasy/dolibarr:${dolibarrVersion}
     docker push tuxgasy/dolibarr:${dolibarrMajor}
+    if [ "${dolibarrVersion}" = ${DOLIBARR_LATEST_TAG} ]; then
+      docker push tuxgasy/dolibarr:latest
+    fi
   fi
 
-  tags="${tags}${dolibarrVersion}"
-  if [ "${dolibarrVersion}" != "develop" ]; then
-    tags="${tags} ${dolibarrMajor}"
+  if [ ${dolibarrVersion} = "develop" ]; then
+    tags="${tags} develop"
+  else
+    tags="${tags} ${dolibarrVersion} ${dolibarrMajor}"
+  fi
```

Comment:
In the generated README, there are double spaces in the tag list. Just remove the space after ${tags}.